blaze-client/src/main/scala/org/http4s/blaze/client/Http1Connection.scala

```diff
@@ -18,27 +18,37 @@ package org.http4s
 package blaze
 package client
 
-import cats.effect.kernel.{Async, Outcome, Resource}
-import cats.effect.std.Dispatcher
 import cats.effect.implicits._
+import cats.effect.kernel.Async
+import cats.effect.kernel.Outcome
+import cats.effect.kernel.Resource
+import cats.effect.std.Dispatcher
 import cats.syntax.all._
 import fs2._
-
-import java.nio.ByteBuffer
-import java.util.concurrent.TimeoutException
-import java.util.concurrent.atomic.AtomicReference
-import org.http4s.Uri.{Authority, RegName}
+import org.http4s.Uri.Authority
+import org.http4s.Uri.RegName
 import org.http4s.blaze.pipeline.Command.EOF
-import org.http4s.blazecore.{Http1Stage, IdleTimeoutStage}
+import org.http4s.blazecore.Http1Stage
+import org.http4s.blazecore.IdleTimeoutStage
 import org.http4s.blazecore.util.Http1Writer
 import org.http4s.client.RequestKey
-import org.http4s.headers.{Connection, Host, `Content-Length`, `User-Agent`}
+import org.http4s.headers.Connection
+import org.http4s.headers.Host
+import org.http4s.headers.`Content-Length`
+import org.http4s.headers.`User-Agent`
 import org.http4s.internal.CharPredicate
-import org.http4s.util.{StringWriter, Writer}
+import org.http4s.util.StringWriter
+import org.http4s.util.Writer
 import org.typelevel.vault._
+
+import java.nio.ByteBuffer
+import java.util.concurrent.TimeoutException
+import java.util.concurrent.atomic.AtomicReference
 import scala.annotation.tailrec
-import scala.concurrent.{ExecutionContext, Future}
-import scala.util.{Failure, Success}
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+import scala.util.Failure
+import scala.util.Success
```

Comment:
Sorry for being late on this, but does flattened imports are actually beneficial?

Reply:
FWIW I like them 👍

Btw big thanks to @bplommer for fixing this up, the messy imports were bugging me!

Reply:
Flattened imports aren’t pretty but they reduce merge conflicts.

Reply:
If it's helping in merging 0.22 -> 0.23 -> main, then it's worth the hassle :+1:

Reply:
I didn't like the expanded imports until I considered the merge angle, and then I liked them a lot.